tests/testthat/test-mod_review_forms.R

```diff
@@ -230,7 +230,7 @@ describe(
         expect_true(app$get_js("document.getElementById('test-review_comment').disabled;"))
         
         # review status and reviewer is saved as expected
-        saved_review_row <- db_get_review(temp_path, subject = "885", form = "Adverse events")
+        saved_review_row <- db_get_review(temp_path, subject_id = "885", item_group = "Adverse events")
```

Comment:
Was the reason of making `db_get_review` general (and not only using `id`) so that this test can stay largely the same?